eth/tools/_utils/hashing.py

```diff
@@ -2,10 +2,15 @@
 
 import rlp
 
+from typing import (
+    Iterable,
+    Tuple,
+)
+
 from eth.rlp.logs import Log
 
 
-def hash_log_entries(log_entries):
+def hash_log_entries(log_entries: Iterable[Tuple[bytes, bytes, bytes]]) -> bytes:
```

Comment:
As a rule of thumb, `keccak` will always return a hash of 32 bytes, hence everything that returns `keccak(x)` should always have the return type `Hash32` rather than just arbitrary bytes.